internal/cli/actions.go

```diff
@@ -396,11 +388,6 @@ func deleteActionCmd(cli *cli) *cobra.Command {
 			}
 
 			return ansi.Spinner("Deleting action", func() error {
-				_, err := cli.api.Action.Read(inputs.ID)
-				if err != nil {
-					return fmt.Errorf("Unable to delete action: %w", err)
-				}
```

Comment:
It didn't make any sense to do a read and then a delete. The delete will throw an error if the action doesn't exist, e.g. 

```
Deleting action... failed

=== example.eu.auth0.com error

 ▸    404 Not Found: That action does not exist.
```